src/qt/locale/bitcoin_it.ts

```diff
@@ -4310,187 +4310,187 @@ p, li { white-space: pre-wrap; }
-        <translation type="unfinished"></translation>
+        <translation>Il presente software è una versione preliminare - ogni uso é a proprio rischio e pericolo. Non utilizzare per il mining o per applicazioni a scopo commerciale</translation>
     </message>
     <message>
         <source>This is the transaction fee you may pay when fee estimates are not available.</source>
-        <translation type="unfinished"></translation>
+        <translation>Questa é la commissione di transazione utilizzata quando il calcolo automatico delle commissioni non é disponibile.</translation>
     </message>
     <message>
         <source>This product includes software developed by the OpenSSL Project for use in the OpenSSL Toolkit %s and cryptographic software written by Eric Young and UPnP software written by Thomas Bernard. Paper wallet art provided by Anacoluthia.</source>
-        <translation type="unfinished"></translation>
+        <translation>Questo prodotto include software sviluppato dall&apos;OpenSSL Project per l&apos;utilizzazione nel Toolkit OpenSSL %s e in software crittografico creato da Eric Young e software UPnP creato da Tomas Bernard. Gli elementi grafici nei portafogli cartacei sono stati creati da Anacoluthia.</translation>
     </message>
     <message>
         <source>Username and hashed password for JSON-RPC connections. The field &lt;userpw&gt; comes in the format: &lt;USERNAME&gt;:&lt;SALT&gt;$&lt;HASH&gt;. A canonical python script is included in share/rpcuser. The client then connects normally using the rpcuser=&lt;USERNAME&gt;/rpcpassword=&lt;PASSWORD&gt; pair of arguments. This option can be specified multiple times</source>
-        <translation type="unfinished"></translation>
+        <translation>Il nome utente e l&apos;hash della password per connessioni JSON-RPC. Il campo &lt;userpw&gt; deve essere nel formato: &lt;NOMEUTENTE&gt;:&lt;SALT&gt;$&lt;HASH&gt;. Uno script python per la generazione del campo é incluso nella cartella share/rpcuser. Il client può successivamente connettersi normalmente utilizzando le opzioni rpcuser=&lt;NOMEUTENTE&gt;/rpcpassword=&lt;PASSWORD&gt;. L&apos;opzione può essere specificata varie volte.</translation>
     </message>
     <message>
         <source>Wallet will not create transactions that violate mempool chain limits (default: %u)</source>
-        <translation type="unfinished"></translation>
+        <translation>Il portafoglio eviterà di generare transazioni che violano i limiti della mempool chain (default: %u)</translation>
     </message>
     <message>
         <source>Warning: The network does not appear to fully agree! Some miners appear to be experiencing issues.</source>
-        <translation type="unfinished"></translation>
+        <translation>Attenzione: La rete sembra essere in disaccordo! Alcuni miner sembrano avere problemi.</translation>
```

Comment:
<translation>Attenzione: La rete sembra essere in disaccordo! Pare che alcuni miner stiano riscontrando dei problemi.</translation>